meetings.md

```diff
@@ -22,31 +23,37 @@ All meetings are expected to have someone in the **Cultivator** &
 
 ### All Hands
 
+📆 **Schedule:** We currently meet every 2 weeks
+🕙 **Duration:** 60 minutes  
+📋 **Template:** [`link.hypha.coop/template`](https://link.hypha.coop/template)
+
 All Hands meetings are our general sync-up. We all try our best to be
 there. We adjust regularity and format of meetings based on time of
 year, member workload (inside and outside co-op), and needs of
 organization-wide initiatives.
 
-📆 **Schedule:** We currently meet every 2 weeks, alternating between
-format of a short **standup** and a **full** meeting.
-
-#### Full
-🕙 **Duration:** 60 minutes  
-📋 **Template:** [`link.hypha.coop/template`](https://link.hypha.coop/template)
-
 This format involves: review of previous action items, working group
 updates, announcements, etc. (See template for specifics.)
 
-#### Standup
-🕙 **Duration:** 15 minutes  
-📋 **Template:** [`link.hypha.coop/standup-template`](https://link.hypha.coop/standup-template)
+#### Annual Members Meeting
+
+📆 **Schedule:** Annually, by our [Bylaws](https://handbook.hypha.coop/bylaws.html#3-member-meetings) they "shall be held within 4 months of the fiscal year end and within 15 months of any previous annual meetings", with appropriate notice given
+🕙 **Duration:** 0.5 day 
+📋 **Template:** 
+
+Members meetings are our annual review where we elect the board, accept finance statements, and hear updates on the work done over the previous year. We tend to schedule this with a board meeting and strategic retreat.
+
+##### Special Members Meeting
```

Comment:
Is the hierarchy of All Hands > Annual > Special what is intended?

Reply:
oop, know I meant it as in the TOC! I'll fix
  - [All Hands](#all-hands)
  - [All Hands: Standup](#standup)
  - [Members Meeting](#annual-members-meeting)
  - [Board Meeting](#board-mee